git.lua

```diff
@@ -142,6 +203,11 @@ local function checkout_spec_generator(token)
         table.insert(res_filter, '*' .. branch)
     end
 
+    for _,branch in ipairs(local_or_remote_branches(token)) do
```

Comment:
By just adding this block you're duplicating local branches in completions table. Consider removal the block above (that responsible for local branches only)